redesign.user.js

```diff
@@ -4464,6 +4485,102 @@ const updateDarkReaderMode = (live = false) => {
 updateDarkReaderMode();
 // endregion
 
+// region Feature: general.quickRoleChange
+if (getSetting('general.quickRoleChange')) {
+    ready(() => {
+        const usermenu = document.getElementById('usermenu-carousel');
+        const usermenuInner = usermenu?.querySelector('.carousel-inner');
+        const roleSelectBtn = usermenu?.querySelector(
+            '.dropdown-item[href*="switchrole.php"]'
+        );
+        if (roleSelectBtn) {
```

Comment:
to avoid an extra level of indentation, how about using
```js
if (!roleSelectBtn) return;
```
instead?

Reply:
Resolved in [55dc979](https://github.com/jxn-30/better-moodle/pull/215/commits/55dc9798a80413e4b9bfc5909bc0e04864e5bcde)